ApiDemos/java/app-v3/src/main/java/com/example/mapdemo/BasicMapDemoActivity.java

```diff
@@ -0,0 +1,53 @@
+/*
```

Comment:
Given these are examples, it makes sense for the source code to be checked-in to version control. Thoughts on appending a "DO NOT EDIT" file header to avoid people accidentally editing this code directly?